gradle.properties

```diff
@@ -15,15 +15,16 @@ project_scm=scm:git:https://github.com/osmlab/atlas-checks.git
 profile=local
 
 ## Local (default)
-checks.local.mainClass=org.openstreetmap.atlas.checks.distributed.IntegrityCheckSparkJob
+checks.local.sharded=true
 checks.local.configFiles=file://@ROOTDIR@/config/configuration.json
-checks.local.input=file://@BUILDDIR@/example/data/pbfs/
+checks.local.input=file://@BUILDDIR@/example/data/atlases/
 checks.local.output=file://@BUILDDIR@/example/data/output/
 checks.local.outputFormats=flags,geojson,metrics
-checks.local.countries=UNK
+checks.local.countries=BLZ
 #checks.local.pbfBoundingBox=
 checks.local.savePbfAtlas=true
 checks.local.compressOutput=false
 checks.local.startedFolder=file://@BUILDDIR@/example/tmp/
 checks.local.master=local
+checks.local.multiAtlas=true
```

Comment:
Why multiAtlas?

Reply:
Because it runs faster locally using multi atlases, and these params are for the local profile.